src/inngest/functions.ts

```
@@ -1,10 +1,21 @@
+import { Agent, gemini, createAgent } from "@inngest/agent-kit";
```

ESLint

[error] 'Agent' is defined but never used.

(@typescript-eslint/no-unused-vars)